radlab-ui/webapp/cypress/e2e/auth/forgot-password.cy.js

```diff
@@ -0,0 +1,49 @@
+/// <reference types="cypress" />
+
+const INVALID_EMAIL = "bademail"
+const FORGOT_EMAIL = "foo@example.com"
+
+describe("RAD Lab UI sign in", () => {
+  beforeEach(() => {
+    // Cypress starts out with a blank slate for each test
+    // so we must tell it to visit our website with the `cy.visit()` command.
+    // Since we want to visit the same URL at the start of all our tests,
+    // we include it in our beforeEach function so that it runs before each test
+    cy.visit("https://rad-lab-ui-d099.uc.r.appspot.com/password-reset/")
```

Comment:
Hardcoded references of the Staging environment.

Reply:
I can extract o an env var. Where would you like the env vars to be defined?
https://docs.cypress.io/guides/guides/environment-variables#Option-1-configuration-file